### PR TITLE
First pass at fixing riak-debug to use cuttlefish

### PR DESCRIPTION
**Pull request by bsparrow435**
This is a hack to get the script working. Still more work to do to integrate cuttlefish fully with the script.

### CODE REVIEW

**Comment by bsparrow435** [comment]
#455

File: rel/files/riak-debug [hunk 1]
```diff
@@ -237,8 +238,8 @@ if [ 0 -ne $(( $get_cfgs + $get_logs + $get_riakcmds + $get_extracmds )) ]; then
     fi
 fi
 
-if [ -f "${riak_etc_dir}"/vm.args ]; then
-    node_name="`egrep '^\-s?name' "${riak_etc_dir}"/vm.args 2>/dev/null | cut -d ' ' -f 2`"
+if [ -f "${riak_etc_dir}"/riak.conf ]; then
+    node_name=`"$riak_bin_dir"/riak config effective | awk '/nodename/ {print $3}'`
```
Comment (seancribbs) [inline rel/files/riak-debug:242]: Couldn't you just read riak.conf directly?

Comment (bsparrow435) [inline rel/files/riak-debug:242]: Yes, the reason i did not was to make the awk script as simple as possible. The .conf can have(or not have) spaces before and after the "=". Calling `riak config effective` guarantees a format where the 3rd element on the line will always be only the value.

Comment (seancribbs) [inline rel/files/riak-debug:242]: Thanks.

**Comment by jaredmorrow** [comment]
What is the status on this PR?  Is it fully reviewed and ready to go in before beta?

**Comment by joedevivo** [comment]
retagging as RC, won't hold up beta for it, but will still try to have it merged for beta

**Comment by jaredmorrow** [comment]
Closing this in favor of #455

**Pull request closed without merging**